x-pack/qa/usage/src/main/java/org/elasticsearch/xpack/usage/UsageQA.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.usage;
+
+import org.elasticsearch.plugins.Plugin;
+
+public class UsageQA extends Plugin {}
```

Comment:
It's not clear to me why we need this. Could you add some javadoc that explains this?

Reply:
Oh this was used by the integration test I removed. Same with the gradle file above. I'll delete them both.

Reply:
Awesome! Thanks, then it makes sense why I couldn't figure out how they were used :p